api-gateway/kong/examples/README.md

```diff
@@ -0,0 +1,225 @@
+# Example
+
+This example is intended to deploy an example using Kong with the `pep-kong` and `jwt-keycloak` plugins and a secure endpoint using a JWT provided by Keycloak.
+
+__ATTENTION__ It's highly recommended to communicate with Keycloak over HTTPS.
+
+
+To run this example, type:
+
+```sh
+docker-compose up
+```
+
+## Using the example
+
+As prerequisites this uses curl and jq .
+
+On Debian-based Linux distributions, you can install these prerequisites by running:
+
+```sh
+sudo apt install curl jq
+```
+
+### Obtaining Access Token
+
+As `user`:
+
+```sh
+JWT=$(curl --location --request POST http://localhost:8000/auth/realms/admin/protocol/openid-connect/token \
+--data-urlencode 'username=user' \
+--data-urlencode 'password=user' \
+--data-urlencode 'client_id=cli' \
+--data-urlencode 'grant_type=password' 2>/dev/null | jq -r '.access_token')
+```
+
+As `admin`:
+
+```sh
+JWT=$(curl --location --request POST http://localhost:8000/auth/realms/admin/protocol/openid-connect/token \
+--data-urlencode 'username=admin' \
+--data-urlencode 'password=admin' \
+--data-urlencode 'client_id=cli' \
+--data-urlencode 'grant_type=password' 2>/dev/null | jq -r '.access_token')
+```
+
+#### Checking if it was defined in the JWT variable
+
+```sh
+echo $JWT
+```
+
+### Accessing resource/endpoint
+
+```sh
+curl -X GET "http://localhost:8000/secure" -H  "Authorization: Bearer ${JWT}"
+```
+
+```sh
+curl -X DELETE "http://localhost:8000/secure" -H  "Authorization: Bearer ${JWT}"
+```
+
+```sh
+curl -X GET "http://localhost:8000/insecure"
+```
+
+## Kong
+
+### Kong settings
+
+In the `kong/kong.config.sh` file are created configurations of routes, services and associations with the kong plugins (pep-kong and jwt-keycloak) for each registered service
+
+### Konga (Kong GUI)
+
+To access the Konga (Kong interface) use http://localhost:1337.
+
+## Keycloak
+
+To access the Admin Keycloak interface use http://localhost:8000/auth and use the login `admin` and password `admin`.
+
+### Keycloak settings
+
+When starting the keycloak, a Realm called **admin** is imported with the configurations listed below:
+
+#### Roles settings
```

Comment:
Are the instructions bellow to reproduce the settings in the imported file?

Reply:
Yes